blueprints/ember-cli-mirage/index.js

```diff
@@ -32,6 +35,23 @@ module.exports = {
       after: '"predef": [\n'
     });
 
+    if (existsSync('tests/helpers/destroy-app.js')) {
+      this.insertIntoFile('tests/helpers/destroy-app.js', '  server.shutdown();', {
+        after: "Ember.run(application, 'destroy');\n"
+      });
+    } else {
+      this.ui.writeLine(
+        EOL +
+        chalk.yellow(
+          '******************************************************' + EOL +
+          'destroy-app.js helper is not present. Please read this' + EOL +
+          'https://gist.github.com/blimmer/35d3efbb64563029505a'   + EOL +
+          'to see how to fix the problem.'                         + EOL +
+          '******************************************************' + EOL
+        )
+      );
```

Comment:
I wanted to make sure that people saw this. Without the callout `*`s it was easy to miss. This is how it looked on my machine:

<img width="454" alt="screen shot 2015-09-20 at 3 43 10 pm" src="https://cloud.githubusercontent.com/assets/630449/9983131/3217165a-5faf-11e5-91c0-483b850c8322.png">